SDWebImage/SDWebImageWebPCoder.m

```diff
@@ -54,11 +55,135 @@ - (BOOL)canIncrementallyDecodeFromData:(NSData *)data {
     return ([NSData sd_imageFormatForImageData:data] == SDImageFormatWebP);
 }
 
+- (SDWebPImage *)webPimageWithData:(NSData *)data {
```

Comment:
This can share a lot of logic with the `decodedImageWithData` method if necessary